Created by Jasper Grimm (UoY)
Update hts_utils.h faidx1_t struct to be compatible with HTSlib 1.9+
 (https://github.com/samtools/htslib/pull/701)
diff -Nru vt-0.57721c/hts_utils.h vt-0.57721d/hts_utils.h
--- vt-0.57721c/hts_utils.h	2021-05-17 14:58:38.000000000 +0100
+++ vt-0.57721d/hts_utils.h	2021-05-17 16:44:20.000000000 +0100
@@ -40,9 +40,11 @@
  *FAI UTILS
  **********/
 typedef struct {
+    int id; // faidx_t->name[id] is for this struct.
     int32_t line_len, line_blen;
     int64_t len;
     uint64_t offset;
+    uint64_t qual_offset;
 } faidx1_t;
 
 KHASH_MAP_INIT_STR(s, faidx1_t)
@@ -52,6 +54,7 @@
     int n, m;
     char **name;
     khash_t(s) *hash;
+    enum fai_format_options format;
 };
 
 /**
